src/components/DatePicker/DatePicker.jsx

```diff
@@ -141,6 +141,7 @@ export const DatePicker = ({
 
     // Comparing `from` and `to` dates hides a weird CSS style when you select the same date twice in a date range.
     const useDateRangeStyle = isRangeVariant && value.from?.getTime() !== value.to?.getTime();
+    const selectedDays = dayjs.isDayjs(value) || isRangeVariant ? value : dayjs(value).toDate();
```

Comment:
dayjs.isDayjs(value) will return Boolean. and in the second statement in OR condition it will return dayjs object. can you explain why this is added ?